app/models/course.rb

```diff
@@ -438,6 +438,18 @@ def project_repos
     self.github_repos.where(is_project_repo: true)
   end
 
+  def instructor
+    Role.where(name: "instructor", resource_id: self.id).first.users.first
```

Comment:
Not in this PR, but as @btk5h suggested, we may want to migrate any code that is doing low level db operations on the `Role` table to instead use API methods of the Rolify gem.

It is possible that in a future upgrade of the Rolify gem, that some of our code that depends on implementation details (such as specific database implementations) could break.

Reply:
This would apply to both regular code and test code.  I'm going to put in a separate issue to address it.